users/views.py

```diff
@@ -177,19 +103,25 @@ def forgot_password(request):
             "message": message,
         }, status=res_status)
 
-
-# TODO: make print the otp in the terminal for now. we will integrate once we get a provider
+@swagger_auto_schema(
+    operation_id='verify_otp',
+    request_body=VerifyOTPSerializer,
+    method='post',
+    responses={
+        '200': set_example(responses.verify_otp_200),
+        '400': set_example(responses.verify_otp_400),
+        '401': set_example(responses.verify_otp_401),
+    },
+)
 @api_view(['POST',])
-@ecell_user 
-@client_check
+@permission_classes([IsAuthenticated])
 def verify_otp(request):
     res_status = status.HTTP_400_BAD_REQUEST
-    user = request.ecelluser
-    print(user)
+    user = request.user
     req_data = request.data
-    print(req_data)
+
     if 'otp' not in req_data:
```

Comment:
I think using a the `VerfyOTPSerializer` snd checking whether it is valis or not would be a better idea.. what do you think ? 😅